src/Symfony/Component/Translation/Bridge/Loco/Tests/LocoProviderFactoryWithoutTranslatorBagTest.php

```diff
@@ -12,8 +13,19 @@
  */
 class LocoProviderFactoryWithoutTranslatorBagTest extends LocoProviderFactoryTest
 {
+    use ExpectDeprecationTrait;
+
     public function createFactory(): ProviderFactoryInterface
     {
         return new LocoProviderFactory($this->getClient(), $this->getLogger(), $this->getDefaultLocale(), $this->getLoader());
     }
+
+    public function testTranslatorBagDeprecation()
+    {
+        $this->expectDeprecation('Since symfony/loco-translation-provider 6.1: Not passing a "Symfony\Component\Translation\TranslatorBagInterface" instance as $translatorBag argument to the "Symfony\Component\Translation\Bridge\Loco\LocoProviderFactory" constructor is deprecated.');
+
+        // Calling $this->createFactory() does not trigger the deprecation, or is not correctly caught by the ExpectDeprecationTrait.
+        // However, manually creating the LocoProviderFactory allow us to test the deprecation.
+        new LocoProviderFactory($this->getClient(), $this->getLogger(), $this->getDefaultLocale(), $this->getLoader());
```

Comment:
If someone knows why using `$this->createFactory()` won't caught the deprecation... :grimacing: